person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
```

Comment:
- [ ] Kindly also include `attr_reader:id ` and define both just after defining class on line 2.